### PR TITLE
Add a GH Actions workflow that builds the dependencies

### PR DESCRIPTION
**Pull request by ricrogz**
This workflow builds all the dependencies required to build sketcher.

### CODE REVIEW

File: schrodinger_build.py [hunk 1]
```diff
@@ -0,0 +1,82 @@
+"""
```
Comment (cdvonbargen) [inline schrodinger_build.py:1]: Meh, let's leave this out until it's something that works.

File: src/app/sketcher_app.cpp [hunk 2]
```diff
@@ -0,0 +1,11 @@
+#include <QApplication>
```
Comment (cdvonbargen) [inline src/app/sketcher_app.cpp:1]: I posted https://github.com/schrodinger/sketcher/pull/3 with all the skeleton code I put together; let's check that in separately to make this review easier on the(my) eyes

File: schrodinger_build.py (hunk 1 above)
Comment (ricrogz) [reply schrodinger_build.py]: Deleted.

File: src/app/sketcher_app.cpp (hunk 2 above)
Comment (ricrogz) [reply src/app/sketcher_app.cpp:1]: ok

**Comment by cdvonbargen** [comment]
Closing out this pull request in favor of https://github.com/schrodinger/sketcher/pull/6

**Pull request closed without merging**